codeHF/PlotEfficiency_recoStep.C

```diff
@@ -0,0 +1,383 @@
+// Computation and plotting of reconstruction efficiency step-by-step
+// Four steps defined: kRecoHFFlag, kRecoTopol, kRecoCand, kRecoPID
```

Comment:
The `k` does not appear in the code.
It would be clearer to describe each step in a couple of words.

Reply:
Done